docs/dev-guides/index.md

```diff
@@ -7,23 +7,34 @@ We're thrilled to have you on board with making the Webaverse a more exciting pl
 
 The Webaverse runs on applications that you can write, package and distribute. These packages are called XRPackages (XRPKs).
 
-We've made it as easy as possible to get started! Please check out our [`XRPackage Overview`](./1-xrpackage-overview.md) to begin your developer journey into the Webaverse.
+We've made it as easy as possible to get started! A few points to start with:
 
-### Supported Asset Types
+- There's no Webaverse SDK! Just make your app using one of the many supported asset types (see below) and quickly package it using our CLI.
+- Once you've made your apps, it's really easy to distribute your packages to decentralized systems -- meaning you don't need your own server or hosting service to get your creations out to the world!
+
+Please check out our [`XRPackage Overview`](./1-xrpackage-overview.md) to begin your developer journey into the Webaverse.
+
+## Supported Asset Types
 
 The Webaverse supports many different asset types to enable creators to easily bring their original content directly into the Webaverse ecosystem.
 
 Currently the following file formats are supported:
 
-- GLTF Models (_.gltf, _.glb)
-- VRM 3D Avatars (.vrm)
-- MagicaVoxel Models (.vox)
-- WebXR sites and applications
+- GLTF Models (`.gltf`, `.glb`)
+- VRM 3D Avatars (`.vrm`)
+- MagicaVoxel Models (`.vox`)
+- WebXR sites and applications (THREE.js, A-Frame, Babylon, etc.)
 - Unity HTML5 Exports (Experimental)
 
-### Import Existing Creations
+## Import Existing Creations
+
+Webaverse can package existing WebXR sites written using THREE.js, A-Frame, Babylon, or any other WebXR compatible JavaScript framework!
+
+For more information on packaging an aug from an existing WebXR site, see the [`Create Your First Aug`](./2-creating-an-aug.md) guide to get started.
```

Comment:
For consistency we should rename this from "aug" to "xrpk", since we changed the main site to use that term.

Reply:
Sure! Would you mind reviewing https://github.com/webaverse/docs/pull/15 first (my PR to move aug -> xrpk) and I can rebase off that and fix any ones leftover?